back/app/serializers/web_api/v1/admin_publication_serializer.rb

```diff
@@ -33,8 +33,8 @@ class WebApi::V1::AdminPublicationSerializer < WebApi::V1::BaseSerializer
     end
   end
 
-  belongs_to :publication, polymorphic: true
-  belongs_to :parent, record_type: :admin_publication
+  belongs_to :publication, polymorphic: true, serializer: :admin_publication
```

Comment:
```suggestion
  belongs_to :publication, polymorphic: true
```

I don't think `serializer` is necessary for polymorphic relationships (when both serializers are in the same namespace).

Reply:
I'm pretty sure it didn't work without this. Probably because it was inferring a serializer name of `PublicationSerializer`, not `AdminPublicationSerializer`.

I'll check again, though.

Reply:
I think it deduces the class from the records themselves and not the name of the attribute when the relationship is polymorphic. Also, come to think of it, the publication is probably either a project or a folder. Shouldn't the serializer be `ProjectSerializer` or `FolderSerializer` instead of `AdminPublicationSerializer`?

Reply:
Not sure. Will investigate.

Reply:
You are correct. It is redundant (and probably wrong).